### PR TITLE
Add burger and webots suggestion

### PR DESCRIPTION
**Pull request by lach-j**
- Adds burger to the list of foods.
- Adds setting up webots as a session suggestion.

### CODE REVIEW

**Review by BaileyJoe — DISMISSED** [review]
👍👍👍

**Review by Happyman31 — DISMISSED** [review]
erry noice

File: NextTime.yaml [hunk 1]
```diff
@@ -19,4 +19,4 @@ suggestions:
   - Markup
   - CMake
   - WSL guide
-
+  - Setting up webots
```
Comment (andrewdeanphillips) [inline NextTime.yaml:22]: Nice suggestion

**Review by andrewdeanphillips — COMMENTED** [review]
Burgers are great

**Comment by andrewdeanphillips** [comment]
> erry noice



> erry noice

yeah true ay

**Review by BaileyJoe — DISMISSED** [review]
looks good

**Review by ysims — APPROVED** [review]
LGTM

**Pull request merged**